README.md

```diff
@@ -19,19 +19,36 @@ Splunk Connect for Kafka is a Kafka Connect Sink for Splunk with the following f
 
 1. Clone the repo from https://github.com/splunk/kafka-connect-splunk
 2. Verify that Java8 JRE or JDK is installed.
-3. Run `bash build.sh`. The build script will download all dependencies and build Splunk Connect for Kafka.
-
-Note: The resulting "splunk-kafka-connect*.tar.gz" package is self-contained. Bundled within it are the Kafka Connect framework, all 3rd party libraries, and Splunk Connect for Kafka.
+3. Run `mvn package`. This will build the jar in the /target directory. The name will be "splunk-kafka-connect-v*.jar".
 
 ## Quick Start
 
 1. [Start](https://kafka.apache.org/quickstart) your Kafka Cluster and confirm it is running.
 2. If this is a new install, create a test topic (eg: `perf`). Inject events into the topic. This can be done using [Kafka data-gen-app](https://github.com/dtregonning/kafka-data-gen) or the Kafka bundle [kafka-console-producer](https://kafka.apache.org/quickstart#quickstart_send).
-3. Untar the package created from the build script: `tar xzvf splunk-kafka-connect-*.tar.gz` (Default target location is /tmp/splunk-kafka-connect-build/kafka-connect-splunk).
-4. Navigate to splunk-kafka-connect directory `cd splunk-kafka-connect`.
-5. Adjust values for `bootstrap.servers` and `plugin.path` inside `config/connect-distributed-quickstart.properties` to fit your environment. Default values should work for experimentation.
-6. Run `./bin/connect-distributed.sh config/connect-distributed-quickstart.properties` to start Kafka Connect.
-7. Run the following command to create connector tasks. Adjust `topics` to set the topic, and  `splunk.hec.token`  to set your HEC token.
+3. Place the jar file created by the maven build (splunk-kafka-connect-v*.jar) in or under the location specified in `plugin.path` (see step 4).
+4. Within Kafka Connect, adjust values for `bootstrap.servers` and `plugin.path` inside `config/connect-distributed.properties` to fit your environment. The following values are also required:
```

Comment:
now that we support all variations of configuration here i don't think we need to explicitly call out all the extra configurables.

Reply:
Removed all except the following:
	internal.key.converter=org.apache.kafka.connect.json.JsonConverter
	internal.value.converter=org.apache.kafka.connect.json.JsonConverter
	internal.key.converter.schemas.enable=false
	internal.value.converter.schemas.enable=false
	offset.flush.interval.ms=10000